vscode-extension/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "wireit-extension",
+  "displayName": "wireit",
+  "description": "For the wireit script runner",
+  "publisher": "todo-register-publisher",
```

Comment:
Where will we publish this? Is there an equivalent to the google org we can use in the vscode extension marketplace? Do we make our own?